### PR TITLE
Changed create function and improved JSDoc

### PR DESCRIPTION
**Pull request by Duddino**
Renamed createFromSeed->create. It now accepts extended public keys as well.
Added some more jsdoc typing

### CODE REVIEW

File: js/pivx_shielding.js [hunk 1]
```diff
@@ -29,19 +44,46 @@ export default class PIVXShielding {
...
+     * @type {Number}
+     * @private
+     */
     this.generatedAddresses = 0;
+    /**
+     * @type {Boolean}
+     * @private
+     */
     this.isTestNet = isTestNet;
-    this.lastBlock = blockHeight;
```
Comment (panleone) [inline js/pivx_shielding.js:37]: Why did you remove the lastBlock? It could be good for consistency check

**Review by panleone — CHANGES_REQUESTED** [review]
tACK apart form 1 comment

File: js/pivx_shielding.js (hunk 1 above)
Comment (Duddino) [reply js/pivx_shielding.js:37]: We aren't using it, if we need in the future we can just re-add this one line

**Review by panleone — APPROVED** [review]
tACK will readd nblock in another PR
